src/components/DocumentationTopic/PrimaryContent/DeclarationSource.vue

```diff
@@ -141,8 +141,9 @@ export default {
         // if we find some text ending with ", " and the next token is the start
         // of a new param, update this token text to replace the space with a
         // newline followed by 4 spaces
-        if (token.text && token.text.endsWith(', ')
-          && nextToken && nextToken.kind === TokenKind.externalParam) {
+        if (token.text && token.text.endsWith(', ') && nextToken
+          && (nextToken.kind === TokenKind.externalParam
+            || nextToken.kind === TokenKind.attribute)) {
```

Comment:
Lots of checks with ANDs and ORs. Maybe we can extract the ORs into a var so we have just ANDs in the `if` body?

Reply:
I agree, it becomes difficult to follow

Reply:
Moved discussion here: https://github.com/apple/swift-docc-render/pull/642#discussion_r1186362051